YadavAkhileshh/Alien-Invasion-Defense
# Missing Favicon

## What does this PR do?

Issue: The project was missing a favicon, which is important for improving user experience and branding. A favicon helps users easily identify the website in their browser tabs and bookmarks.

Proposed Changes: In this update, I have designed and implemented a visually appealing favicon that reflects the brand identity of the website. This enhancements will significantly improve the website's usability and contribute to a more polished and professional appearance.

## Fixes #201 

##Images/Videos:
![Screenshot 2024-10-09 231155](https://github.com/user-attachments/assets/8e1b7f29-1eaa-4b6a-a4cb-c73e51f5a689)


## Type of change
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How should this be tested?

1. Open the project folder on your local machine.
2. Navigate to the index.html file in the project directory.
3. Open the index.html file using a live server extension in your code editor (such as Live Server for Visual Studio Code):
    Right-click on index.html.
    Select “Open with Live Server.”
4. Observe the browser tab: Note that there is no favicon displayed before the changes.
5. Check the bookmarks: Attempt to bookmark the site and observe the absence of a favicon in the bookmarks list.

## Mandatory Tasks
